### PR TITLE
Fix carriage return in sed command in syncnipt script

### PR DESCRIPTION
**Pull request by barrystokman**
This PR fixes the use of `^M` in a `sed` command in the script `syncnipt.bash`. `^M` stands for carriage return, in the sample sheet a carriage return needs to be replaced with a new line `\n`, and for some reason the symbol `^M` can end up as an actual hard return in the code 😡 

This [version](https://semver.org/) is a:
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by barrystokman** [comment]
@karlnyr @henrikstranneheim I'm not waiting for approval and reviews for this, my apologies.

W9hWZGq

**Comment by henrikstranneheim** [comment]
Had a feeling you would say that - Just made it in time then 😄

**Comment by barrystokman** [comment]
Bumped:
image
Deployed:
image

**Pull request merged**
